.dev/tests/phpunit/includes/test-coblocks-block-assets.php

```diff
@@ -98,7 +98,31 @@ public function test_editor_assets_scripts() {
 		$this->go_to( '/wp-admin/post-new.php' );
 		$this->coblocks_block_assets->editor_assets();
 
-		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered ) );
+		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered,  ) );
+
+		$path = 'wp-content/plugins/coblocks/dist/coblocks-*.js';
```

Comment:
```suggestion
		$path  = dirname( __FILE__ ) . '/../../../../dist/coblocks-*.js';
```

Reply:
Looking at the unit tests action it looks like these were running on GitHub but not running locally. With the path fix above it should work locally and on GitHub.

**Github Actions:**
> Tests: 178, Assertions: 366, Skipped: 3.

**Locally:**
> Tests: 178, Assertions: 351, Skipped: 3.